FirebaseDatabaseTestingSupport.podspec

```diff
@@ -0,0 +1,57 @@
+Pod::Spec.new do |s|
+  s.name                    = 'FirebaseDatabaseTestingSupport'
+  s.version                 = '1.0.0'
+  s.summary                 = 'Firebase SDKs testing support types and utilities.'
+
+  s.description      = <<-DESC
+  Type declarations and utilities needed for unit testing of the code dependent on Firebase SDKs
```

Comment:
Make sure to add info that the podspecs are just there for testing. See https://github.com/firebase/firebase-ios-sdk/commit/397215b9f304ba320c7d18c9839509535d8624e1#diff-221380d0cac6c2109a93596f5086bc8691024208216ff59d18546a8162162550R7